server/ai_live_video.go

```diff
@@ -583,10 +583,19 @@ func (s *SlowOrchChecker) GetCount() int {
 
 func LiveErrorEventSender(ctx context.Context, streamID string, event map[string]string) func(err error) {
 	return func(err error) {
-		GatewayStatus.Store(streamID, map[string]interface{}{
-			"last_error":      err.Error(),
-			"last_error_time": time.Now().UnixMilli(),
-		})
+		status, ok := GatewayStatus.Get(streamID)
```

Comment:
I know this PR is just a draft but wanted to mention that this method of get-and-set is not thread safe and could result in different errors stomping on top of one another.

Reply:
Yep thanks, I just wanted to quickly try something on staging, I think I will go with this method though just with a thread safe function. See what you think 🙏